### DISCUSSION TITLE
How to set up VS Code project from GitHub in StackBlitz

### DISCUSSION THREAD

**Discussion by pawsie** [stackblitz/core#1946] [Q&A]
Hi, I have a working VS Code project in my GitHub repo. When I sync it over to my StackBlitz project, the dependencies window appears for a while, then disappears. The status then stays forever at "Compiling application and starting dev server...".

https://github.com/pawsie/wordleVS
https://stackblitz.com/edit/angular-ivy-fyf8jw

How do I set up the dependencies for compiling in StackBlitz?

Thanks!

**Comment by kc0tlh** [comment]
Try it out in [our new VS Code editor](https://stackblitz.com/codeflow)!
